.circleci/config.yml

```diff
@@ -5,9 +5,10 @@ jobs:
       - image: valhalla/docker:build-2.5.0
     steps:
       - checkout
-      - run: apt-get update && apt-get -y install clang-tidy-5.0 clang-5.0 curl git
+      - run: apt-get update && apt-get -y install clang-tidy-5.0 clang-5.0 curl git nodejs
       - run: ./scripts/format.sh && ./scripts/error_on_dirty.sh
       - run: ./.circleci/make_test.sh && run-clang-tidy-5.0.py -header-filter "^$(pwd)/(src|valhalla/(baldr|midgard|sif|odin|thor|skadi|tyr|loki|mjolnir)/.*" && ./scripts/error_on_dirty.sh
+      - run: nodejs ./scripts/validate_changelog.js
```

Comment:
wouldn't you want faster formatting type operations to be earlier in CI so that it doesnt have to do the whole build just to fail on formatting in th changelog?

Reply:
Ah good point! :+1: